src/onegov/feriennet/views/activity.py

```diff
@@ -409,8 +410,18 @@ def view_activities(self, request):
 
     filters = {k: v for k, v in filters.items() if v}
 
+    all_sponsors = layout.app.banners(request)
+    main_sponsor = all_sponsors[0]
+    sponsors = all_sponsors[1:len(all_sponsors)]
+
+    activities = self.batch if show_activities else None
+    activities = [a for a in activities]
```

Comment:
This would raise `TypeError: 'NoneType' object is not iterable` if show_activities is False. Better something like:

     activities = list(self.batch) if show_activities else []